clang/lib/Parse/ParseOpenACC.cpp

```diff
@@ -232,32 +235,87 @@ void ParseOpenACCClauseList(Parser &P) {
     P.Diag(P.getCurToken(), diag::warn_pragma_acc_unimplemented_clause_parsing);
 }
 
-void ParseOpenACCDirective(Parser &P) {
-  OpenACCDirectiveKind DirKind = ParseOpenACCDirectiveKind(P);
+} // namespace
+
+// Routine has an optional paren-wrapped name of a function in the local scope.
+// We parse the name, emitting any diagnostics
+ExprResult Parser::ParseOpenACCRoutineName() {
+
+  ExprResult Res;
+  if (getLangOpts().CPlusPlus)
+    Res = ParseCXXIdExpression(/*isAddressOfOperand=*/false);
+  else {
```

Comment:
Enclose in braces for uniformity

Reply:
Interestingly, when I was rewriting our brace policy a few years ago, nearly this exact case was one that people wanted to preserve, so the rule essentially became "once you start, use it for the rest, but not before you need to".  

I wasn't a huge proponent of it, and don't mind here, so I've added them.